smif/convert/area.py

```diff
@@ -63,12 +64,12 @@ class RegionRegister(object):
     between data values relating to compatible sets of regions.
     """
     def __init__(self):
-        self._register = {}
+        self._register = OrderedDict()
         self._conversions = defaultdict(dict)
```

Comment:
Yes, would be more space-efficient not to hold onto the regions - though this is a bigger change.

Currently we hold on to all the region sets while building the conversion register so that adding a region triggers the generation of the conversions to/from the new region and each previously-registered region. 

Could change the API so we have to register each conversion with knowledge of the pair of region sets? Or could introduce a builder-type step that holds onto all regions while setting up, and outputs the more parsimonious representation that the converter needs.